plugin_runner/plugin_runner.py

```diff
@@ -267,6 +272,15 @@ def HandleEvent(self, request: EventRequest, context: Any) -> Iterable[EventResp
                         f" {event.context['path']}"
                     )
                     effect_list = [Response(status_code=HTTPStatus.INTERNAL_SERVER_ERROR).apply()]
+            if event.type == EventType.SIMPLE_API_WEBSOCKET_AUTHENTICATE:
+                if len(relevant_plugin_handlers) == 0:
+                    effect_list = [DenyConnection().apply()]
+                elif len(relevant_plugin_handlers) > 1:
+                    log.error(
+                        f"Multiple handlers responded to {EventType.Name(EventType.SIMPLE_API_WEBSOCKET_AUTHENTICATE)}"
+                        f" {event.context['channel']}"
+                    )
+                    effect_list = [DenyConnection().apply()]
```

Comment:
I appreciate denial by default, but what if multiple handlers respond, but only one provides an ACCEPT or DENY effect? Example: I have one handler that controls whether a subscription should be accepted, but I have another handler that hits an HTTP API whenever that happens to log an audit trail.

Reply:
looks like for SimpleAPI requests just above we have the same logic, if multiple handlers then we respond with an internal server error

Reply:
Hmm, well, I'd prioritize consistency over handling my edge case. I retract my comment!